lib/commands/web.js

```diff
@@ -131,28 +131,6 @@ commands.title = async function title () {
   return await this.executeAtom('title', [], true);
 };
 
-commands.getCookies = async function getCookies () {
-  if (!this.isWebContext()) {
-    throw new errors.NotImplementedError();
-  }
-
-  this.log.debug('Retrieving all cookies');
-
-  let script = 'return document.cookie';
-  let jsCookies = await this.executeAtom('execute_script', [script, []]);
-
-  let cookies = [];
-  try {
-    for (let [name, value] of _.toPairs(cookieUtils.createJWPCookie(undefined, jsCookies))) {
-      cookies.push({name, value});
-    }
-    return cookies;
-  } catch (err) {
-    this.log.error(err);
-    throw new errors.UnknownError(`Error parsing cookies from result: '${jsCookies}'`);
-  }
-};
-
 commands.setCookie = async function setCookie (cookie) {
```

Comment:
it also makes sense to move this helper to cookies.js module

Reply:
it would also be interesting to know if the remote debugger has some helpers to set cookies

Reply:
Yea, probably can. I found https://github.com/WebKit/WebKit/blob/main/Source/JavaScriptCore/inspector/protocol/Page.json#L191-L196, so am thinking to play something later

Reply:
as another pr